ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -33,14 +33,18 @@ else
   exit 1
 fi
 
-NUTANIX_OS_ENDPOINT="${CLUSTER_PROFILE_DIR}/nutnaix_os_endpoint"
-if [[ ${NUTANIX_OS_ENDPOINT} == "" ]]; then
-  echo "Did not find nutanix os endpoint, exit now"
-  exit 1
+if [[ -f ${CLUSTER_PROFILE_DIR}/secrets.sh ]]; then
+  NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+else
+  NUTANIX_AUTH_PATH=/var/run/vault/nutanix/secrets.sh
 fi
+declare nutnaix_os_endpoint
+# shellcheck disable=SC1090
+source "${NUTANIX_AUTH_PATH}"
+nutanix_region_endpoint="https://"${nutnaix_os_endpoint}
```

Comment:
I think can directly update cluster-profile vault key to resolve it.

Reply:
Yes, if job test passed, i will ask @sgaoshang to update vault key and delete the line.